.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,28 @@
+<!--- Provide a general summary of your changes in the Title above -->

```

Comment:
For both of these, can we make them just very simple? In my opinion, we just need to ask for a screenshot if it is applicable to your PR or issue. Other than that our issues and PRs are pretty simple.

Reply:
Sure thing. 👍